### PR TITLE
Define PROXY_TAG in istio.VERSION

### PR DESCRIPTION
**Pull request by chxchx**
This is required to automate release now that proxy is part the release artifacts.

**Release note**:

```release-note
None
```

### CODE REVIEW

File: install/updateVersion.sh [hunk 1]
```diff
@@ -53,6 +54,7 @@ while getopts :gi:n:p:x:c:sA:P:E: arg; do
     p) PILOT_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
     x) MIXER_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
     c) CA_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
+    r) PROXY_TAG="${OPTARG}";;
```
Comment (sebastienvas) [inline install/updateVersion.sh]: you need to add r in the getopts as well

Comment (chxchx) [reply install/updateVersion.sh]: Good catch

**Comment by chxchx** [comment]
PTAL

**Comment by sebastienvas** [comment]
@chxchx Could you resolve your conficts ?

**Comment by ldemailly** [comment]
side note: there is no real value in using the -flags in updateVersion, it's much simpler to edit istio.VERSION first and then run it with no flag

maybe we can just get rid of all those flags (in 0.3)

**Comment by chxchx** [comment]
poking submit queue

**Pull request merged**